plugin/core/sessions.py

```diff
@@ -155,6 +155,9 @@ def get_initialize_params(variables: Dict[str, str], workspace_folders: List[Wor
                     "documentationFormat": ["markdown", "plaintext"],
                     "tagSupport": {
                         "valueSet": completion_tag_value_set
+                    },
+                    "resolveSupport": {
+                        "properties": ["detail", "documentation", "additionalTextEdits", "command"]
```

Comment:
Maybe it doesn't make sense to specify `detail` and `documentation` since we resolve only after completing so we can't make use of those anyway.

Reply:
> Maybe it doesn't make sense to specify detail and documentation

For me also it doesn't make sense because only those two properties were allowed to be lazy loaded prior the 3.16 LSP spec release. So specifying the in the `resolveSupport` might seem redundant.

> we resolve only after completing so we can't make use of those anyway.

But I do not agree 100% with this one. We do resolve the completion item now in two cases.
- when we select the completion item(that is implemented with this PR)
- when we click the `More` link in the AC panel. (We resolve the completion item and grab the `detail` and `documentation ` of the resolved completion item, and that is the reason why I would also specify them in the `resolveSupport`). 

I feel like being explicit and leaving  `detail` and `documentation` in `resolveSupport` could not hurt us. 
What do you think?

Reply:
Alright.

Reply:
Here's what vscode does: https://github.com/microsoft/vscode-languageserver-node/blob/d58c00bbf8837b9fd0144924db5e7b1c543d839e/client/src/common/client.ts#L1598-L1600

Reply:
Should I remove the `command`?

Reply:
Yeah, let's keep it at most as complicated as vscode :)

Reply:
I guess that it wont hurt. 

The reason on why I also added the command is that we use  "additionalTextEdits", "command" after we resolve the completion item. So for our use case it would have been ok to announce that we support lazy loading for the `command` also 

https://github.com/sublimelsp/LSP/pull/1651/files#diff-1e31f37d39e534dca2dca7fd81d29d7e5a039a4b55a5765fdb4292c7a87760aaR116

https://github.com/sublimelsp/LSP/pull/1651/files#diff-1e31f37d39e534dca2dca7fd81d29d7e5a039a4b55a5765fdb4292c7a87760aaR120

But I will remove it.

Reply:
843dcf6